src/test_typing_extensions.py

```diff
@@ -6402,6 +6402,34 @@ def test_typevartuple(self):
         class A(Generic[Unpack[Ts]]): ...
         Alias = Optional[Unpack[Ts]]
 
+    @skipIf(
+        sys.version_info < (3, 11, 1),
+        "Not yet backported for older versions of Python"
+    )
```

Comment:
I verified locally that these tests pass with Python 3.11.1 but not with Python 3.11.0...

Reply:
It seems like the failures are just that we're missing the defaults from `__args__`, which doesn't seem too bad. (It may cause problems for introspection but shouldn't usually break creation of types that are normally in annotations.) I'm fine leaving those out for now.

Reply:
Yes, the test failures "just" indicate incorrect behaviour rather than crashes